HolyshiftService/JwtService.cs

```diff
@@ -0,0 +1,96 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+public class JwtService
+{
+    private readonly byte[] _keyBytes = new byte[32];
+    private readonly string _issuer;
+    private readonly string _audience;
+    private readonly int _accessTokenExpirationMinutes;
+    private readonly int _refreshTokenExpirationDays;
+    private readonly ILogger<JwtService> _logger;
+
+    public JwtService(IConfiguration configuration, ILogger<JwtService> logger)
+    {
+        var jwtSettings = configuration.GetSection("JwtSettings");
+        var key = jwtSettings["Key"];
```

Comment:
The key is 100% not `jwtSettings`.

The key stored in the settings is the serialized form not the binary form. So you need some kind deserialize it from base 64 string or hex string.

> I prefer base64
> The conversion should be done at the caller not in the constructor.

Reply:
I just noticed that you don't actually use this field in the first place.

Reply:
yes I am not using it that's just a part of an old code I tried , so I just remove it right?